CSharp/BatchExplorer/Models/CertificateModel.cs

```diff
@@ -0,0 +1,113 @@
+﻿using GalaSoft.MvvmLight.Messaging;
+using Microsoft.Azure.Batch;
+using Microsoft.Azure.Batch.Common;
+using Microsoft.Azure.BatchExplorer.Helpers;
+using Microsoft.Azure.BatchExplorer.Messages;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.BatchExplorer.Models
+{
+    public class CertificateModel : ModelBase
+    {
+        public CertificateModel(Certificate source)
+        {
+            this.Certificate = source;
+        }
+
+        public string Thumbprint
```

Comment:
Hmm, looks like other entities do, so I guess so. Strangely, it seems to work fine without the attribute, but I'll follow the convention just in case.